app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
```

Comment:
a esto solemos llamarlo directamente con el nombre del recurso en plural
```suggestion
        render json: notes, status: :ok, each_serializer: IndexNoteSerializer
```

Reply:
Hola Flor, agregué capturas para los casos tristes, una para cuando el tipo de nota no es válido y otra para cuando no existe un id.
Corregí los nombres de los métodos para las respuestas de la api:):
![image](https://github.com/ross-garcia/Training-Rails-UGUITO-API-Bootstrap/assets/160248891/97db2d9a-fd24-48e7-8560-461c520b9495)